lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -383,19 +389,19 @@ static BuiltinInst *simplifyOperands(const SILTensorOpInfo &opInfo,
   // rewritten - either to load an address argument or expand a struct
   // parameter.
   auto canSimplifyOperand =
-      [&](SILType type, GraphOperationInfo::OperandClass operandClass) -> bool {
+      [&](SILType type, GraphOperationInfo::OperandLowering lowering) -> bool {
     return isLoadableAddressType(type) ||
            getPrimitiveStructField(type.getASTType()) != nullptr ||
-           operandClass == GraphOperationInfo::OperandClass::Out;
+           lowering == GraphOperationInfo::OperandLowering::Out;
   };
 
-  // If we don't have to change any operands, don't rewrite the builtin.
+  // If we don't have to change any operands, don't rewrite the graph_op.
   bool mustChangeBuiltin = false;
```

Comment:
nit: change this flag's name while you are at it?

Reply:
changed